src/simplex.cpp

```diff
@@ -181,7 +181,7 @@ uint16_t snoise16(uint32_t x, uint32_t y) {
     // Add contributions from each corner to get the final noise value.
     // The result is scaled to return values in the interval [-1,1].
     int32_t n = n0 + n1 + n2;    // .30
-    n = (n << 6) / 46360; // fix scale to fit exactly in an int16_t
+    n = ((n >> 8) * 23163) >> 16; // fix scale to fit exactly in an int16
```

Comment:
I guess you meant `int16_t`?

```suggestion
    n = ((n >> 8) * 23163) >> 16; // fix scale to fit exactly in an int16_t
```

Same for 3D noise (which was also wrong in my original FastLED PR).